app/scripts/modules/amazon/src/securityGroup/configure/configSecurityGroup.mixin.controller.js

```diff
@@ -169,7 +169,7 @@ module.exports = angular
         }
       }
 
-      const match = (available || []).find(vpc => vpc.ids.includes($scope.securityGroup.vpcId));
+      const match = (available || []).find(vpc => vpc.label === $scope.securityGroup.vpcName);
```

Comment:
Using the vpn name here to ensure that it is selected if its available for the region and account.